### PR TITLE
chore(medusa,core-flows): update request return items and shipping

### PR DESCRIPTION
**Pull request by carlos-r-l-rodrigues**
What:
* `POST /admin/returns/:id/shipping-methods/:action_id`
* `POST /admin/returns/:id/request-items/:action_id`

CLOSES: CC-184, CC-185

### CODE REVIEW

File: packages/core/types/src/workflow/order/request-item-return.ts [hunk 1]
```diff
@@ -9,6 +10,15 @@ export interface DeleteRequestItemReturnWorkflowInput {
   action_id: string
 }
 
+export interface UpdateRequestItemReturnWorkflowInput {
+  return_id: string
+  action_id: string
+  data: {
+    quantity?: BigNumberInput
+    internal_note?: string | null
```
Comment (fPolic) [inline packages/core/types/src/workflow/order/request-item-return.ts:18]: question: should we be able to update return reason here?

**Review by fPolic — APPROVED** [review]
LGTM 👍

File: packages/core/types/src/workflow/order/request-item-return.ts (hunk 1 above)
Comment (carlos-r-l-rodrigues) [reply packages/core/types/src/workflow/order/request-item-return.ts:18]: yes, good point.
I have to add it to the request item as well

Comment (carlos-r-l-rodrigues) [reply packages/core/types/src/workflow/order/request-item-return.ts:18]: I will them it in a next PR because multiple files are changed...

**Pull request merged**